policy/modules/system/locallogin.te

```diff
@@ -124,9 +124,10 @@ term_relabel_unallocated_ttys(local_login_t)
 term_relabel_all_ttys(local_login_t)
 term_setattr_all_ttys(local_login_t)
 term_setattr_unallocated_ttys(local_login_t)
-term_relabel_all_ptys(local_login_t)
-term_setattr_generic_ptys(local_login_t)
 term_use_usb_ttys(local_login_t)
+term_use_all_ptys(local_login_t)
+term_relabel_all_ptys(local_login_t)
+term_setattr_all_ptys(local_login_t)
```

Comment:
Yes, I would say same as above but i don't know why it was already allowed access to all pty's

So these and the ones for getty and init as fine with me this way.